packages/ava/src/data/statistics/common.ts

```diff
@@ -0,0 +1,264 @@
+import { assert } from '../utils';
```

Comment:
The file should be `rename` instead of `new`. And, we name it `base` is better than now one.

Reply:
![image](https://user-images.githubusercontent.com/71261480/225201950-13f7d416-a543-460c-99e9-ed02d109195d.png)